cmd/omegaup-grader/frontend_handler_test.go

```diff
@@ -118,7 +118,7 @@ func newInMemoryDB(t *testing.T, partialScore bool) *sql.DB {
 			alias varchar(32) NOT NULL UNIQUE,
 			scoreboard int NOT NULL DEFAULT '1',
 			points_decay_factor double NOT NULL DEFAULT '0',
-			partial_score tinyint(1) NOT NULL DEFAULT '1',
+			score_mode enum('partial','all_or_nothing','max_per_group') NOT NULL DEFAULT 'partial',
```

Comment:
```suggestion
			score_mode varchar NOT NULL DEFAULT 'partial',
```

esto no es sqlite válido.